src/components/shop/scrollable-list/scrollable-list-buttons.js

```diff
@@ -42,6 +41,8 @@ export const createListButton = (scene, item, title, action, parent) => {
     const ariaLabel = `${item.state ? item.state + " " : ""}${item.title} - ${item.subtitle}`;
     const channel = buttonsChannel(scene);
     const group = scene.scene.key;
+    const disabled = item.state && scene.config.states[item.state] ? scene.config.states[item.state].disabled : false;
+    defaults.key = disabled ? "itemBackgroundLocked" : "itemBackground";
```

Comment:
rather than add a new asset, would this work?
`disabled && gelButton.off(Phaser.Input.Events.POINTER_OVER);`

Reply:
its technically the same asset file just only using 1 frame of it but yea i guess that's simpler, will have a look